docs/standard/datetime/resolve-ambiguous-times.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How to resolve ambiguous times
+description: How to resolve ambiguous times
+keywords: .NET, .NET Core
+author: stevehoag
+manager: wpickett
+ms.date: 08/16/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: e86050c6-d16d-405e-8bba-7205945c9a81
+---
+
+# How to: resolve ambiguous times
+
+An ambiguous time is a time that maps to more than one Coordinated Universal Time (UTC). It occurs when the clock time is adjusted back in time, such as during the transition from a time zone's daylight saving time to its standard time. When handling an ambiguous time, you can do one of the following:
+
+* Make an assumption about how the time maps to UTC. For example, you can assume that an ambiguous time is always expressed in the time zone's standard time.
+
+* If the ambiguous time is an item of data entered by the user, you can leave it to the user to resolve the ambiguity.
+
+This article shows how to resolve an ambiguous time by assuming that it represents the time zone's standard time.
+
+## To map an ambiguous time to a time zone's standard time
+
+1. Call the [System.TimeZoneInfo.IsAmbiguousTime(DateTime)](xref:System.TimeZoneInfo.IsAmbiguousTime(System.DateTime)) or [System.TimeZoneInfo.IsAmbiguousTime(DateTimeOffset)](xref:System.TimeZoneInfo.IsAmbiguousTime(System.DateTimeOffset)) method to determine whether the time is ambiguous.
+
+2. If the time is ambiguous, subtract the time from the [TimeSpan](xref:System.TimeSpan) object returned by the time zone's 'BaseUtcOffset' property.
```

Comment:
why not link to the property here? topic on msdn links to [System.TimeZoneInfo.BaseUtcOffset](https://docs.microsoft.com/en-us/dotnet/core/api/system.timezoneinfo#System_TimeZoneInfo_BaseUtcOffset)